spec/index.bs

```diff
@@ -1920,6 +1922,29 @@ interface GPUAdapterInfo {
         this value is not recommended. Applications which change their behavior based on the
         {{GPUAdapterInfo}}, such as applying workarounds for known driver issues, should rely on the
         other fields when possible.
+
+    : <dfn>properties</dfn>
+    ::
+        {{GPUFeatureName|Feature-specific}} values of the [=adapter=].
+</dl>
+
+<script type=idl>
+interface GPUAdapterProperties {
+    readonly attribute unsigned long minSubgroupSize; // From "subgroups" feature
+    readonly attribute unsigned long maxSubgroupSize; // From "subgroups" feature
+};
+</script>
+
+<dl dfn-type=attribute dfn-for=GPUAdapterProperties data-timeline=content>
+    : <dfn>minSubgroupSize</dfn>
+    ::
+        If the {{GPUFeatureName/"subgroups"}} feature is supported, the minimum
```

Comment:
Are there known limits on the values? (between 4 and 128 most likely). What are values reported when the feature isn't present on the adapter? It's not clear at the moment that these are undefined if the subgroup feature isn't there.

Reply:
I see two possibilities:
1. make the defaults 4 and 128
2. return undefined if the feature is not supported

I lean towards 1 as the cleaner solution.

Reply:
I like 1 if we ~know those are always going to be the bounds.

Technically couldn't we return real values here even if the subgroups feature isn't supported, at least on some devices? I wonder if applications could care about it for performance reasons even when not using subgroup operations.

Reply:
I went with 1 for now and made unsupported values the same as default values (4 and 128). We could return real values in some unsupported cases, but I'm not sure it'd be worth it.